### PR TITLE
[Imagery Plugin] Prev/Next functionality

### LINKED ISSUE #3209: Imagery should include Next and Previous image controls
### Overview
The main image in Image Telemetry should include "Prev" and "Next" arrow buttons. User request from VIPER dsims.

Screen Shot 2020-07-16 at 4 54 29 PM

### Details
- Clicking the Prev or Next arrows should display the image immediately preceding or following the the current image, respectively. This should have _exactly_ the same effect as clicking the the thumb immediately preceding the image that the user is currently on. 
  - Clicking Prev should add the "paused" state to the image interface - again, same as clicking a previous thumb does now.
  - If the image is currently in the "paused" state and clicks the Next button such that they would display the most current image available, the interface should be released from the "paused" state; yet again, same as clicking the most current thumb.
- If there are no images in the current data query preceding or following the current image, the relevant arrow should have the `disabled` attribute applied to it.
- Arrows buttons should be available everywhere that the main image appears: in the main view, in Layouts, in large view.
- Arrow buttons will be hidden until the user hovers over the main image.

Related: #3302

### PR DESCRIPTION
**Pull request by jvigliotta**
Added Previous and Next arrow functionality to the imagery plugin. This enables the users (in different views) to navigate forward and backward in the list of current images.

closes #3209 

## Author Checklist
Changes address original issue? Yes
Unit tests included and/or updated with changes? ( #3328 )
Command line build passes? Yes
Changes have been smoke-tested? Yes
Testing instructions included? Yes

### CODE REVIEW

File: src/plugins/imagery/components/ImageryViewLayout.vue [hunk 1]
```diff
@@ -262,6 +275,47 @@ export default {
...
+                disabled = true;
+            }
+
+            return disabled;
+        },
+        isPrevDisabled() {
+            let disabled = false;
+            let index = this.selectedImageIndex();
+
+            if (index === 0) {
```
Comment (nikhilmandlik) [inline src/plugins/imagery/components/ImageryViewLayout.vue:314]: Also need to disable this when `imageHistory` length is `0` or `1`

**Comment by nikhilmandlik** [comment]
> @charlesh88 Move the position of the prev/next arrows to be centered close together, to allow easy A/B image scrubbing amongst two close images. Observed from driving dSim 8/14/20.

Wondering if it is better to add keyboard arrow shortcuts to this as well.

**Comment by charlesh88** [comment]
> > @charlesh88 Move the position of the prev/next arrows to be centered close together, to allow easy A/B image scrubbing amongst two close images. Observed from driving dSim 8/14/20.
> 
> Wondering if it is better to add keyboard arrow shortcuts to this as well.

YES, that's a great idea. Unless that's super easy, maybe file as a separate enhancement.

**Comment by nikhilmandlik** [comment]
> separate enhancement +1

File: src/plugins/imagery/components/ImageryViewLayout.vue (hunk 1 above)
Comment (jvigliotta) [reply src/plugins/imagery/components/ImageryViewLayout.vue]: added a check for less than 2!

**Comment by jvigliotta** [comment]
> > separate enhancement +1

Added #3302 for the arrow key functionality!

**Review by nikhilmandlik — COMMENTED** [review]
LGTM

**Review by nikhilmandlik — APPROVED** [review]

**Review by charlesh88 — APPROVED** [review]
Backing off the re-positioning of the arrow buttons in favor of doing via keyboard arrow keys instead. So, LGTM!

**Review by nikhilmandlik — APPROVED** [review]
I said LGTM 😅  (trying Dave's Approach)

**Comment by nikhilmandlik** [comment]
### Reviewer Checklist
Check | Passed
-- | --
Changes appear to address issue? | Y
Appropriate unit tests included? | Y
Code style and in-line documentation are appropriate? | Y
Commit messages meet standards? | Y
Has associated issue been labelled unverified? (only applicable if this PR closes the issue) | Y

**Pull request merged**